GitUI/Avatars/InitialsAvatarProvider.cs

```diff
@@ -123,10 +123,11 @@ private static (string? name, char[]? separator) NameSelector(string? name, stri
         private readonly Color[] _avatarColors =
         {
             Color.RoyalBlue,
-            Color.DarkRed,
+            Color.SlateGray,
             Color.Purple,
-            Color.ForestGreen,
-            Color.DarkOrange
+            Color.OrangeRed,
+            Color.Teal,
+            Color.OliveDrab,
```

Comment:
I had a lot of difficulties to find colors that are dark enough (to be able to see initials in white) and different enough to be easily distinguished without ending up with something horrible.

But if you have a lot of time to spend on that, I will be happy to change it.
Maybe it's easy to add a hidden setting for that. I will have a look...

Reply:
I am not good at selecting colors, better at complaining...
The colors are not that different, really just the SlateGray that was added and that happen to be on quite a lot of the example...
Maybe removing SlateGray is an option, keep it for now.

Reply:
> Code in c# a way to get composite color so no matter the background color a text color will show.
>
> To get a composite color that will contrast with any given background color, you can use the following C# code:
>
> scssCopy code
>
> Color GetContrastColor(Color backgroundColor) { // Calculate the luminance of the background color double luminance = (0.2126 * backgroundColor.R + 0.7152 * backgroundColor.G + 0.0722 * backgroundColor.B) / 255; // Use a threshold value to determine whether to use black or white as the foreground color Color foregroundColor = (luminance > 0.5) ? Color.Black : Color.White; return foregroundColor; }
>
> This method takes a Color object representing the background color and returns a Color object representing the foreground color that will provide the best contrast with the background.
>
>The method works by calculating the luminance of the background color using the standard formula for converting RGB values to grayscale. If the luminance is greater than 0.5, it means the background color is relatively light, so the method returns black as the foreground color. If the luminance is less than or equal to 0.5, the background color is relatively dark, so the method returns white as the foreground color.
>
>You can use this method in your code by calling GetContrastColor() with the background color you want to contrast with. For example:
>
> scssCopy code
> 
> Color backgroundColor = Color.FromArgb(255, 128, 0); // orange Color foregroundColor = GetContrastColor(backgroundColor); Console.WriteLine($"Foreground color: {foregroundColor}");
>
>This code will output "Foreground color: Color [A=255, R=255, G=255, B=255]" because white is the best foreground color to use with an orange background

Colors are complex and require tuning. I would have expected that there were more guidelines and libs supporting this.
Still amazed by the work done by @NikolayXHD for the dark theme, so sad to loose it.
This would be even harder with another theme.

Reply:
i have pushed a commit so that there is a (hidden) setting for the colors list. Like that every one could remove the Grey 😉